hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystem.java

```diff
@@ -86,6 +90,13 @@ public void initialize(URI uri, Configuration conf) throws IOException {
         KeyPair keyPair = getKeyPair(conf);
         KeyStorageStrategy keyStore = new FileKeyStorageStrategy(delegate, keyPair);
 
+        if (Crypto.isNativeCodeLoaded()) {
```

Comment:
with the idea being that we could write a unit to make that this logging works as intended?

Definitely seems useful, both for this and in general. I'll investigate.